api/bolt/migrator/migrator.go

```diff
@@ -37,6 +38,7 @@ type (
 		userService            *user.Service
 		versionService         *version.Service
 		fileService            portainer.FileService
+		tagService             *tag.Service
```

Comment:
Relocate under `userService` to follow alphabetic order in this file. All the services of this package should follow this rule, services from outside this package can be put at the end of object definition (like the `portainer.FileService`).